aten/src/ATen/TensorSubclassLikeUtils.h

```diff
@@ -28,8 +28,7 @@ constexpr auto kFunctorchWrappedTensors = DispatchKeySet({
 
 constexpr auto kTensorSubclassLike = kFunctorchWrappedTensors | DispatchKeySet({
     DispatchKey::Batched,
-    DispatchKey::SparseCPU,
-    DispatchKey::SparseCUDA,
+    DispatchKey::Sparse,
     DispatchKey::SparseCsrCPU,
     DispatchKey::SparseCsrCUDA,
```

Comment:
We'll likely need to detach CPU/CUDA from CSR dispatch keys as well, similar to `Sparse`:
```suggestion
    DispatchKey::SparseCsr,
```

Reply:
I'll probably do that in a followup. For context: There actually isn't a hard requirement on which way to represent `SparseCsr` - it's more about whether or not we want to trade off dispatch key slots for space in the runtime operator table. We could either:

(1) Make it a "per-backend functionality key". There would be one dispatch key for `SparseCsr`, and any backend (CPU, CUDA, etc) could implement their own kernels for it separately. Advantage: we save some key space, and give every backend the flexibility to override its meaning (instead of being limited to just CPU and CUDA). Minor disadvantage: we blow up the size of the runtime operator table a bit, since we now need to store an extra (# backends) number of slots for SparseCsr, for every operator.

(2) Leave the existing `SparseCsrCPU/CUDA` keys, which saves some runtime operator table space.

It sounds like we should eventually do (1)